java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -0,0 +1,63 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.BitcountOptions;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "Bitmap Commands" group for standalone and cluster
+ * clients.
+ *
+ * @see <a href="https://redis.io/docs/latest/commands/?group=bitmap">Bitmap Commands</a>
+ */
+public interface BitmapBaseCommands {
+    /**
+     * Counts the number of set bits (population counting) in a string. Returns zero if the key is
+     * missing as it treated as an empty string.
+     *
+     * @see <a href="https://redis.io/commands/bitcount/">redis.io</a> for details.
+     * @param key The key to count set bits of.
+     * @return The number set bits in the string.
+     * @example
+     *     <pre>{@code
+     * Long payload = client.bitcount("myKey1").get();
+     * assert payload == 2L; // Count of set bits in a string
+     * }</pre>
+     */
+    CompletableFuture<Long> bitcount(String key);
+
+    /**
+     * Counts the number of set bits (population counting) in a string. Returns zero if the key is
```

Comment:
reminder for the same thing as the comment above

Reply:
Sorry can you clarify for which comment?

Reply:
All docs should be same, please, align that (and in transaction too)